lib/iterable_basics.dart

```diff
@@ -237,6 +253,25 @@ extension NumIterableBasics<E extends num> on Iterable<E> {
         ? this.reduce((a, b) => (a + b) as E)
         : this.fold(0, (prev, element) => prev + addend(element));
   }
+
+  /// Returns the average of all the values in this iterable.
+  ///
+  /// If [addend] is provided, it will be used to compute the value to be
```

Comment:
Can you rename `addend` to `value` here as well?

Reply:
Sorry, I overlooked this one. I updated it.